ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
I'll throw a counter in so that it raises an Exception if the motor hasn't started after 1s, same for wait_for_stop but maybe with a longer timeout there.

I needed this when I was working on MINDCUB3R (will submit that in the next few weeks, I need to rebuild it). None of the run_XYZ() calls block so if I say run_to_rel_pos(180) and need to take some action once it reaches 180 I need to know that A the motor started moving and B it stopped.  I don't remember the exact scenario but there were times when I got buggy behavior prior to implementing wait_for_running().